src/Glpi/Form/AccessControl/FormAccessControl.php

```diff
@@ -264,6 +265,48 @@ public function createConfigFromUserInput(array $input): JsonConfigInterface
         return $strategy->createConfigFromUserInput($input);
     }
 
+    /**
+     * Encode the input name to make sure it is unique and multiple items
+     * can be updated using a single form.
+     *
+     * Content must be decoded using `splitEncodedInputs`.
+     *
+     * @param string $name
+     * @return string
+     */
+    public function encodeInputName(string $name): string
```

Comment:
`encode` is prone to confusion. I first thought its purpose was made to encode HTML special chars.

```suggestion
    public function getNormalizedInputName(string $name): string
```